spacy_udpipe/udpipe.py

```diff
@@ -0,0 +1,202 @@
+import re
+from typing import Dict, List, Optional, Union
+
+from ufal.udpipe import (InputFormat, Model, OutputFormat, ProcessingError,
+                         Sentence)
+
+from .utils import get_path
+
+
+class PretokenizedInputFormat(object):
```

Comment:
I'd suggest to add tokenizer to name.

Reply:
As the short doc says, this is just a dummy tokenizer. From `v2.0` of spaCy, `Tokenizer.tokens_from_list` has been deprecated and [`Doc.__init__`](https://spacy.io/api/doc#init) is the new method to create a `Doc` from pretokenized text. However, this early in the pipeline, not enough info is available to justify creating a `Doc` so `PretokenizedInputFormat` used as a dummy tokenizer is imho the most elegant solution because it also follows the API of other [`UDPipe Tokenizers`](http://ufal.mff.cuni.cz/udpipe/api-reference#input_format_new_input_format).